Docs/source/usage/parameters.rst

```diff
@@ -1988,7 +1988,7 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
 
 * ``<diag_name>.fields_to_plot`` (list of `strings`, optional)
     Fields written to output.
-    Possible scalar fields: ``part_per_cell`` ``rho`` ``phi`` ``F`` ``part_per_grid`` ``divE`` ``divB`` and ``rho_<species_name>``, where ``<species_name>`` must match the name of one of the available particle species. Note that ``phi`` will only be written out when do_electrostatic==labframe.
+    Possible scalar fields: ``part_per_cell`` ``rho`` ``phi`` ``F`` ``part_per_grid`` ``divE`` ``divB`` and ``rho_<species_name>``, where ``<species_name>`` must match the name of one of the available particle species. Note that ``phi`` will only be written out when do_electrostatic==labframe. Also, note that for ``<diag_name>.diag_type = BackTransformed``, the only scalar field currently supported is ``rho``.
```

Comment:
Just a quick question: why is `rho_<species>` not supported as well? :)

Reply:
`rho` is obtained by performing Lorentz transform of `j` and `rho`
https://github.com/ECP-WarpX/WarpX/blob/1f3d2f3678f5ae2e934f1531bfcad4a2aae10b3f/Source/Diagnostics/ComputeDiagFunctors/BackTransformFunctor.cpp#L217 
We can add an enhancement once we have particles in the buffers and compute rho per species

But, currently there are some issues with the particle buffer -- fixed here
https://github.com/ECP-WarpX/WarpX/pull/3056

Reply:
Ok, thanks for the details :)